services/bwdserver-deployment-darklang/bwdserver-deployment.template.yaml

```diff
@@ -0,0 +1,362 @@
+            timeoutSeconds: 10
+            periodSeconds: 5
+          readinessProbe: # can it serve http requests?
+            httpGet:
+              path: /k8s/readinessProbe
+              port: 11002
+            initialDelaySeconds: 0
+            periodSeconds: 5
+            successThreshold: 3
+          livenessProbe: # is it still alive?
+            httpGet:
+              path: /k8s/livenessProbe
+              port: 11002
+            initialDelaySeconds: 0
+            periodSeconds: 10
+            timeoutSeconds: 10
+            failureThreshold: 3 # kill container after 30 seconds (3x10s checks)
+
+          # ---------------------
+          # Environment
+          # ---------------------
+          envFrom:
+            - configMapRef:
+                name: "{VERSIONED-CONFIGMAP:app-config}"
+          env:
+            # rollbar
+            - name: DARK_CONFIG_ROLLBAR_POST_SERVER_ITEM
+              valueFrom:
+                secretKeyRef:
+                  name: rollbar-account-credentials
+                  key: post_token
+            # pusher
+            - name: DARK_CONFIG_PUSHER_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: pusher-account-credentials
+                  key: key
+            - name: DARK_CONFIG_PUSHER_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: pusher-account-credentials
+                  key: secret
+            # database (sql server in the same pod)
+            - name: DARK_CONFIG_DB_HOST
+              value: 127.0.0.1
+            - name: DARK_CONFIG_DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: cloudsql-db-credentials
+                  key: username
+            - name: DARK_CONFIG_DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: cloudsql-db-credentials
+                  key: password
+            # honeycomb
+            - name: DARK_CONFIG_HONEYCOMB_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: honeycomb-account-credentials
+                  key: api-key
+
+
+        #########################
+        # Legacyserver container
+        #########################
+        - name: legacy-ctr
+          image: "gcr.io/balmy-ground-195100/legacyserver:{IMAGEID:legacyserver}"
+          ports:
+            - name: legacy-port
+              containerPort: 5000
+
+          # ---------------------
+          # Security - https://docs.bridgecrew.io/docs
+          # ---------------------
+          imagePullPolicy: Always
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - NET_RAW # https://docs.bridgecrew.io/docs/bc_k8s_27
+            privileged: false
+            # this is expected to run as root. We're getting rid of the container
+            # soon so not worth fixing.
```

Comment:
Can you elaborate here?
Is it _currently_ running as root and we want to not run as root? and the comment here is that it's not worth fixing the existing/old container?

Reply:
This is copied from the old config. I did a pass to make it as secure as possible a while back, including getting as many containers as possible not running as root. However, legacyserver expects to run as root because we run everything in ocaml using `sudo`.